api/api/models/event_ingest_model.py

```diff
@@ -7,7 +7,7 @@
 MAX_EVENTS_PER_REQUEST = 100
 
 
-class EventsIngestModel(Body):
+class EventIngestModel(Body):
 
     def __init__(self, events: Optional[list] = None) -> None:
         """EventsIngestModel body model.
```

Comment:
Remove the extra "S" in the constructor comment

```suggestion
        """EventIngestModel body model.
```